tests/integration/tier0/check-user-response/test_user_response.py

```diff
@@ -0,0 +1,50 @@
+from envparse import env
+
+
+def test_check_user_response(convert2rhel):
+
+    # Run c2r registration with no username and password provided
+    # check for user prompt enforcing input, then continue with registration
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {}".format(
+            env.str("RHSM_SERVER_URL"),
+        )
+    ) as c2r:
+        c2r.expect_exact(" ... activation key not found, username and password required")
+        c2r.expect_exact("Username: ")
+        c2r.sendline()
+        if c2r.expect_exact(["Username", "Password"]) == 0:
+            # Provide username, expect password prompt
+            c2r.sendline(env.str("RHSM_USERNAME"))
+        else:
+            c2r.sendcontrol("d")
+        c2r.expect_exact("Password: ")
+        c2r.sendline()
+        if c2r.expect_exact(["Password", "Enter number of the chosen subscription: "]) == 0:
+            # Provide password, expect successful registration and subscription prompt
+            c2r.sendline(env.str("RHSM_PASSWORD"))
+        else:
+            c2r.sendcontrol("d")
+        c2r.expect_exact("Enter number of the chosen subscription: ")
+        c2r.sendcontrol("d")
+    assert c2r.exitstatus != 0
+
+    # Run c2r registration with activation key provided
+    # check for user prompt while organization left blank
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {} -k {}".format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_KEY"),
+        )
+    ) as c2r:
+        c2r.expect_exact(" ... activation key detected: ")
+        c2r.expect_exact("Organization: ")
+        c2r.sendline()
+        substitute_org = "foo"
+        if c2r.expect_exact("Organization: ") == 0:
+            c2r.sendline(substitute_org)
+        else:
+            c2r.sendcontrol("d")
+        c2r.expect_exact("Registering the system using subscription-manager ...")
+        c2r.sendcontrol("d")
+    assert c2r.exitstatus != 0
```

Comment:
Hey, @danmyway, just a question... It's not easier to separate this into another test case? like 

```python
def test_check_user_response_user_and_password():
   ...
   
def test_check_user_response_organization(): 
   ...
```